hw/xbox/nv2a/gl/gloffscreen_cgl.c

```diff
@@ -21,21 +21,21 @@
  * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
  * THE SOFTWARE.
  */
-#include <stdlib.h>
+#include <dlfcn.h>
 #include <stdio.h>
+#include <stdlib.h>
 #include <string.h>
-#include <dlfcn.h>
 
 #include "gloffscreen.h"
 
-#include <OpenGL/CGLTypes.h>
 #include <OpenGL/CGLCurrent.h>
+#include <OpenGL/CGLTypes.h>
 
 struct _GloContext {
-  CGLContextObj     cglContext;
+    CGLContextObj cglContext;
 };
 
-/* Create an OpenGL context for a certain pixel format. formatflags are from 
+/* Create an OpenGL context for a certain pixel format. Format flags are from
```

Comment:
Why did you touch this?
What does the comment even refer to? I only see "format attributes" below.

Also nit while we are at it: Some comments start lowercase (like the next one below).

Reply:
Because it looks like a typo, is it not?

Reply:
I literally have no idea what the comment even wants to express to be honest. I can't find the respective code to match this comment.

But to me it read like `formatflags` was actually a variable / field or type of some sort (implying it's not a typo).
The comment below also doesn't even use "format flags". It only refers to "pixel format attributes".